templates/user.tmpl

```diff
@@ -0,0 +1,208 @@
+{{define "title"}}Internal: User ({{ .User.Firstname }} {{if ne .User.Firstname .User.Nickname}}({{ .User.Nickname }}) {{end}}{{ .User.Lastname }}){{end}}
+{{define "content"}}
+    <div class="column is-10" style="min-height: 88vh">
+        <section class="hero is-info welcome is-small">
+            <div class="hero-body">
+                <div class="container">
+                    <h1 class="title">{{ .User.Firstname }} {{if ne .User.Firstname .User.Nickname}}({{ .User.Nickname }}) {{end}}{{ .User.Lastname }}</h1>
+                </div>
+            </div>
+        </section>
+        <br>
+        <div class="columns box" style="height: fit-content">
+            <div class="column is-2">
+                <figure class="avatar">
+                    <img src="{{.User.Avatar}}" alt="{{.User.Avatar}}" width="128px"
+                                                 height="128px"/>
+                </figure>
+                <div class="buttons" style="display: block">
+                    {{if eq .User.DeletedBy.UserID -1}}
+                        <a class="button is-info is-outlined" onclick="resetPasswordModal()">
+                            Reset password
+                        </a>
+                        <a class="button is-info is-outlined" onclick="editUserModal()">
+                            Edit
+                        </a>
+                        {{if .User.Enabled}}
+                            <a class="button is-warning is-outlined" onclick="disableUserModal()">
+                                Disable
+                            </a>
+                        {{else}}
+                            <a class="button is-warning is-outlined" onclick="enableUserModal()">
+                                Enable
+                            </a>
+                        {{end}}
+                        <a class="button is-danger is-outlined" onclick="deleteUserModal()">
+                            Delete
+                        </a>
+                    {{end}}
+                </div>
+            </div>
+            <div class="column">
+                <p id="message" style="color: green"></p>
+                <p id="error" style="color: red"></p>
+                {{parseUserIntoHTML .User .UserPermissions}}
+            </div>
+        </div>
+    </div>
+    {{if not .User.DeletedAt.Valid}}
+        {{template "modals" .}}
+    {{end}}
+{{end}}
+
+{{define "modals"}}
+    <div id="resetPasswordModal" class="modal">
+        <div class="modal-background"></div>
+        <div class="modal-content">
+            <div class="box">
+                <article class="media">
+                    <div class="media-content">
+                        <div class="content">
+                            <p class="title">Are you sure you want to reset the password?</p>
+                            <p><strong>This action cannot be undone</strong><br>An email will be sent to the user for
```

Comment:
Fairly sure this can be undone - just reset it again. (see earlier comment about my general thoughts on these notes, this being a good example)

Reply:
Reset password will force the user to change their password and will heck for if it is the same i think

Reply:
At that point they can just not do the reset - my point is that whether something can or can't be undone is a very vague definition with a lot of caveats.

Reply:
Do you suggest just removing the undone part?

Reply:
Yeah, I'm not sure what its benefit is relative to the potential for confusion. (Not just here, but on the other templates as well.)

Reply:
I'll remove all references to it